R/get_commits.R

```diff
@@ -4,11 +4,7 @@
 #' @param gitstats_obj  A GitStats object.
```

Comment:
I would consider removing wrappers for basic functionalities as the wrapping masterclass is now simplified and it could be easily called on it's own